docs/support/managing-ckeditor-logo.md

```diff
@@ -22,46 +22,46 @@ To remove the logo, you need to obtain a commercial license and then configure t
 
 Refer to the {@link support/license-key-and-activation License key and activation} guide for details on where to find the license key and how to use it in your configuration.
 
-## How to configure the layout of the "Powered by CKEditor" logo?
+## How to configure the "Powered by CKEditor" logo?
 
 For open source, free users, the "Powered by CKEditor" logo will always be displayed. There is, however, some degree of control over it.
```

Comment:
I don't like the sound of "some degree of" in this context, however I'm unable to come up with anything better 😞 So I'll leave as it is.

Reply:
"can control some aspects of it"?